clash-lib/prims/verilog/Clash_Explicit_Testbench.primitives.yaml

```diff
@@ -17,7 +17,7 @@
       always @(~IF~ACTIVEEDGE[Rising][0]~THENposedge~ELSEnegedge~FI ~ARG[3]) begin
         if (~ARG[6] !== ~ARG[7]) begin
           $display("@%0tns: %s, expected: %b, actual: %b", $time, ~LIT[5], ~ARG[7], ~ARG[6]);
-          $finish;
+          $finish(0);
```

Comment:
Are you sure you want to change this? If you're intending to set the exit code you can't in standard verilog, but some simulators (like `iverilog`) _may_ have extensions for this (e.g. `finish_and_return(0)` in `iverilog` is `exit(0)`).

What you've actually changed is the level of information printed when stopping simulation to be explicitly "print no information". The other options you can give according to the standard being

```
0: prints nothing
1: prints simulation time and location
2: prints simulation time, location and statistics about the used memory and CPU time
```

Reply:
I tried to resolve a rebase conflict (but messed up):

https://github.com/clash-lang/clash-compiler/commit/07073588403ff39291447a68c7b0bf8891e10f53

Reply:
F

Reply:
Fixed!